templates/js/sso-tab-with-obo-flow/teamsapp.local.yml.tpl

```diff
@@ -55,7 +56,7 @@ provision:
       # Relative path to this file. Environment variables in manifest will
       # be replaced before apply to AAD app
       manifestPath: ./aad.manifest.json
-      outputFilePath: ./build/aad.manifest.${{TEAMSFX_ENV}}.json
+      outputFilePath : ./build/aad.manifest.${{TEAMSFX_ENV}}.json
```

Comment:
There's an extra space here.

Reply:
There might be an extra space in the template for the `aadAppUpdate` action.

Reply:
Updated.